mode/examples/Topics/File IO/LoadFile2/LoadFile2.pyde

```diff
@@ -0,0 +1,51 @@
+"""
+LoadFile 2
+
+This example loads a data file about cars. Each element is separated
+with a tab and corresponds to a different aspect of each car. The file stores 
+the miles per gallon, cylinders, displacement, etc., for more than 400 different
+makes and models. Press a mouse button to advance to the next group of entries.
+"""
+
+from Record import *
+
+RECORDS = []
+RECORD_COUNT = 0
+NUM = 9  # Display this many entries on each screen.
+STARTING_ENTRY = 0  # Display from this entry number
+
+def setup():
+    global RECORDS, RECORD_COUNT
```

Comment:
why CAPITAL LETTERS?

Reply:
Was this in reaction to pylint "Invalid name for a constant"? If so perhaps see for example...

https://softwareengineering.stackexchange.com/questions/342374/should-i-really-use-all-uppercase-for-my-constants

Reply:
No, I would prefer to reserve CAPS for constants, which these aren't.

Reply:
Sorry, that's what I meant by the link -- pylint mistakenly recommends that global variables be formatted as constants when they aren't constants -- either ignore the bad recommendation or reorganize.

> Pylint assumes that code that actually does the work will be inside a function. Because you have this code at the top level of your code instead of inside a function it gets confused.

Reply:
I haven't seen any pylint output in this request.

Reply:
I've misinterpreted the PEP8 global constant recommentadtion!